### PR TITLE
Self review project

### PR DESCRIPTION
**Pull request by maen-al-khraisha**
in this task make a self review of my own project

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -1 +1,86 @@
...
+}
+#title{
+    z-index: 2;
+    position: relative;
+    color: #fff;
+    margin-top: 30px;
+}
+div{
+    margin: 20px 0;
+}
```
Comment (maen-al-khraisha) [inline css/style.css:44]: hey, I think adding a general style selector to the top of the CSS file will be a better practice.

File: css/style.css [hunk 2]
```diff
@@ -1 +1,86 @@
...
+    margin: 15px 0;
+}
+button{
+    padding: 15px 15px;
+    margin: auto;
+    border: none;
+    width: 99%;
+    background-color: rgb(9, 122, 122);
+    color: #fff;
+}
```
Comment (maen-al-khraisha) [inline css/style.css:86]: hey :),
Check linter report and fix the issues.😃

File: index.html [hunk 3]
```diff
@@ -1 +1,102 @@
...
+    </title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Raleway:wght@100;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <h1 id="title">freeCodeCamp Survey Form</h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
```
Comment (maen-al-khraisha) [inline index.html:17]: good structure, 
I think adding these two items inside a section will be more readable and easy to maintain.  😑

**Review by maen-al-khraisha — COMMENTED** [review]
good job, 
please review the comment and make the necessary changes, please. 👍

**Review by Wes-Isaac — APPROVED** [review]
Hi @maen1980 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by maen-al-khraisha** [comment]
Hi Isaac,
Thank you :)
    On Tuesday, February 15, 2022, 10:59:03 AM GMT+2, Isaac ***@***.***> wrote:  
 
 


@Wes-Isaac approved this pull request.

Hi @maen1980 ,

Your project is complete! There is nothing else to say other than... it's time to merge it 
Congratulations! 🎉
 
Cheers and Happy coding! 👏 👏 👏
 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Pull request merged**